integration-webcert/src/main/java/se/inera/intyg/minaintyg/integration/webcert/converter/data/value/IcfValueConverter.java

```diff
@@ -17,14 +18,23 @@ public CertificateDataValueType getType() {
 
   @Override
   public CertificateQuestionValue convertToValue(CertificateDataElement element) {
-    final var value = ((CertificateDataIcfValue) element.getValue()).getText();
-    // TODO: Handle icf-codes
-    return getText(value);
+    final var icfCodes = ((CertificateDataIcfValue) element.getValue()).getIcfCodes();
+    final var text = ((CertificateDataIcfValue) element.getValue()).getText();
+
+    if (icfCodes != null) {
+      final var collectionsLabel = ((CertificateDataConfigIcf) element.getConfig()).getCollectionsLabel();
+      final var value = collectionsLabel + "\n" + icfCodes + "\n" + "\n" + text;
```

Comment:
Could this be turned into a method to better describe the reason for doing this? Maybe something like formatValue or something else more describing?